src/test-run/index.js

```diff
@@ -199,34 +236,16 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     this.pendingRequest = null;
 
-    if (!this.running)
-        this._start();
-
-    var driverStatus                  = msg.status;
-    var shouldRejectPendingDriverTask = driverStatus.pageError &&
-                                        this.pendingDriverTask &&
-                                        isCommandRejectableByPageError(this.pendingDriverTask.command);
-
-    if (shouldRejectPendingDriverTask)
-        this._rejectPendingDriverTask(driverStatus.pageError);
-    else {
-        this.pendingJsError = this.pendingJsError || driverStatus.pageError;
+    // NOTE: driver send repeated status if it didn't get the response for the first one.
+    // It's possible when the page was unloaded after driver sent the status.
+    if (msg.status.id === this.lastDriverStatusId && this.lastDriverStatusResponse)
+        return this.lastDriverStatusResponse;
```

Comment:
In this case we fill `this.lastDriverStatusResponse` when `executeCommand` is called [here](https://github.com/DevExpress/testcafe/pull/529/files#diff-a21f56bee759db22062e32250e3df48cR161)